smriprep/workflows/outputs.py

```diff
@@ -731,7 +731,7 @@ def init_anat_derivatives_wf(
                               ('source_files', 'source_file')]),
         (name_regs, ds_regs, [('hemi', 'hemi')]),
         (inputnode, name_reg_fsLR, [('sphere_reg', 'in_file')]),
```

Comment:
Yes. I don't think it will, but conceivably it could swap L/R labels.
```suggestion
        (inputnode, name_reg_fsLR, [('sphere_reg_fsLR', 'in_file')]),
```